vortexor/src/main.rs

```diff
@@ -53,12 +54,19 @@ pub fn main() {
     let max_streams_per_ms = value_t_or_exit!(matches, "max_streams_per_ms", u64);
     let exit = Arc::new(AtomicBool::new(false));
     // To be linked with the Tpu sigverify and forwarder service
-    let (tpu_sender, _tpu_receiver) = unbounded();
+    let (tpu_sender, tpu_receiver) = unbounded();
```

Comment:
Given that vortexor lives on a different host, would be nice to see some mechanism to control the max size of this queue in case the connection gets stuck. Are you sure we can not use a bounded channel here?

Reply:
This was based on the current sig-verify code which uses unbounded channel. I do not think the vortexor on different changes natures regarding this. Sigverfiy is packets from this channel and do the verifications. I think it it changes it may actually be more performant. But I agree with you from defensive programming aspects, if we cannot guarantee the receiver is always faster the the  sender, it is good practice to guard against excessive memory usage which could happen under load.

Reply:
Will address in a flow-on PR. @alexpyattaev

Reply:
It is not just about memory usage. It is more about the whole fact that if vortexor OOMs under some network delay/loss conditions, figuring out why it happens will be tough, which may deter operators from using it.